.bazelrc

```diff
@@ -28,10 +28,6 @@ build --features swift.index_while_building
 # Prevents leaking unexpected binaries via PATH to tests
 build --test_env=PATH=/usr/bin:/bin:/usr/sbin:/sbin
 
-# Enable device deps when the build matches the arm64 simulator
-# Bazel 5 - matches ios_sim_arm64 CPU, Bazel 4 - needs "override" flags
-build --features apple.arm64_simulator_use_device_deps
```

Comment:
I moved this to the arm64 CI job where it's tested. This way we test without it by default to ensure we catch issues without this flag enabled.